Silicon/NXP/iMX6Pkg/Drivers/LcdifGop/Lcdif.c

```diff
@@ -0,0 +1,273 @@
+/** @file
+*
+*  Copyright (c) 2019 Microsoft Corporation. All rights reserved.
+*
+*  This program and the accompanying materials
+*  are licensed and made available under the terms and conditions of the BSD License
+*  which accompanies this distribution.  The full text of the license may be found at
+*  http://opensource.org/licenses/bsd-license.php
+*
+*  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+*
+**/
+
+#include <Uefi.h>
+#include <Library/IoLib.h>
+#include <Library/TimerLib.h>
+#include <Library/DebugLib.h>
+
+#include <iMXDisplay.h>
+#include <iMX6ClkPwr.h>
+
+#include "Lcdif.h"
+
+#if HDMI_I2C
+//
+// Board specific implementation
+//
+VOID LcdifBoardConfigureDisplay (
+    IMX_DISPLAY_TIMING* Timing,
+    VOID* FrameBuffer
+    );
+#endif // HDMI_I2C
+
+EFI_STATUS
+LcdifInitDisplay (
+    IMX_DISPLAY_TIMING* Timing,
+    VOID* FrameBuffer,
+    UINT32 LcdifBase
+    )
+{
+    volatile UINT32 value32;
+    DEBUG ((DEBUG_INFO, "LcdifInitDisplay\n"));
+
+    // TODO: add timeouts to initialization loops
+
+    //
+    // Reset LCD controller (dancing bits reset procedure)
+    //
+    MmioWrite32(
```

Comment:
The EDK2 style guidelines require all function and macro calls to have a space after the name.